packages/web-api/src/types/request/files.ts

```diff
@@ -144,6 +144,8 @@ export type FileUploadV2 = FileUpload & {
   alt_text?: string;
   /** @description Channel ID where the file will be shared. If not specified the file will be private. */
   channel_id?: string;
+  /** @deprecated use channel_id instead */
```

Comment:
Not sure if we could do anything self then deprecate this 🤔

Reply:
Nailed it!